### ISSUE TITLE
Version 0.7.4 broke travis build 

### ISSUE THREAD

**Issue by dilawar** [matthew-brett/delocate#40]
The logs of build are here: https://api.travis-ci.org/v3/job/435219821/log.txt 

I checked locally and could reproduce the error with python3.7. It failed due to following. Note sure if older version will also break with python3.7 .

```
ImportError: cannot import name 'open_for_csv' from 'wheel.util' (/Users/dilawars/Library/Python/3.7/lib/python/site-packages/wheel/util.py)
```

**Comment by anthrotype** [comment]
master branch has a fix for that and other issues introduced by 0.32.0
https://github.com/matthew-brett/delocate/commit/68355ee937826e0e5db4c75833f165db670e6ba4

Until new delocate is released and the dust settles I think it's better for you to pin wheel to 0.31.0

**Comment by hugovk** [comment]
For the cause see:
* https://github.com/pypa/wheel/issues/255
* https://github.com/matthew-brett/multibuild/issues/202

**Comment by matthew-brett** [comment]
I think you're picking up an old version of delocate - see the logs - you've got 0.7.3 ... You should not see the error with the new 0.7.4.

**Comment by dilawar** [comment]
> I think you're picking up an old version of delocate - see the logs - you've got 0.7.3 ... You should not see the error with the new 0.7.4.

I checked locally with 0.7.4 and 0.7.3 . Same error with both. Now I am using the older version of wheel as suggest by @anthrotype ; worked fine locally. Trying on travis now.

**Comment by matthew-brett** [comment]
@dilawar - could you post logs of your local testing?  Here is me testing delocate 0.7.4 in a fresh virtualenv:

```
(test3) [brettmz@colles-12146 ~]$ pip install delocate
Collecting delocate
  Using cached https://files.pythonhosted.org/packages/09/53/a3c5680d223d4f8076da3ae646c8d7c36b70b9f027c482c75234db60e44c/delocate-0.7.4.tar.gz
Requirement already satisfied: wheel in ./.virtualenvs/test3/lib/python3.6/site-packages (from delocate) (0.32.0)
Building wheels for collected packages: delocate
  Running setup.py bdist_wheel for delocate ... done
  Stored in directory: /Users/brettmz/Library/Caches/pip/wheels/6f/60/4c/1ea0f3631bf8074eeaa6dae0be38b81150520a6301298e4235
Successfully built delocate
Installing collected packages: delocate
Successfully installed delocate-0.7.4
(test3) [brettmz@colles-12146 ~]$ python -c 'from delocate import delocate_wheel, __version__'
(test3) [brettmz@colles-12146 ~]$ pip install wheel==0.31.1
Collecting wheel==0.31.1
  Using cached https://files.pythonhosted.org/packages/81/30/e935244ca6165187ae8be876b6316ae201b71485538ffac1d718843025a9/wheel-0.31.1-py2.py3-none-any.whl
Installing collected packages: wheel
  Found existing installation: wheel 0.32.0
    Uninstalling wheel-0.32.0:
      Successfully uninstalled wheel-0.32.0
Successfully installed wheel-0.31.1
(test3) [brettmz@colles-12146 ~]$ python -c 'from delocate import delocate_wheel, __version__'
(test3) [brettmz@colles-12146 ~]$ 
```

This should also be tested in the delocate Travis-CI setup, but maybe I'm not doing it right.

**Comment by ezralanglois** [comment]
I see this issue also on travis:

`Collecting delocate
  Using cached https://files.pythonhosted.org/packages/09/53/a3c5680d223d4f8076da3ae646c8d7c36b70b9f027c482c75234db60e44c/delocate-0.7.4.tar.gz
    Complete output from command python setup.py egg_info:
    error in delocate setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Expected version spec in machomachomangler; sys_platform == 'win32' at ; sys_platform == 'win32'`

**Comment by matthew-brett** [comment]
Ah yes - probably you're on Python 3.4?   To fix, you need to update setuptools...

**Comment by ezralanglois** [comment]
Building Python 2.7.11

**Comment by matthew-brett** [comment]
Yes, still probably too-old setuptools.

**Comment by ezralanglois** [comment]
Yes, I can confirm that worked. Thanks!

**Comment by dilawar** [comment]
Got fixed after deprecating wheel version to 0.31.0. Using latest `delocate` now. https://api.travis-ci.org/v3/job/437511577/log.txt

**Comment by dilawar** [comment]
You are right. 0.7.4 works with wheel 0.32.1 (released yesterday). I havn't tested with 0.32.0 .
